xnmt/translator.py

```diff
@@ -37,17 +37,11 @@ def calc_loss(self, src, trg):
     '''
     raise NotImplementedError('calc_loss must be implemented for Translator subclasses')
 
-  def translate(self, src):
-    '''Translate a particular sentence.
-
-    :param src: The source, a sentence or a batch of sentences.
-    :returns: A translated expression.
+  def calc_reinforce_loss(self, reward):
```

Comment:
Can we come up with a more general name? We might not necessarily be using reinforce, but still want to use this capability.